### PR TITLE
[stable/sumologic-fluentd] Update api versions to support k8s 1.16

### PR DESCRIPTION
**Pull request by antongorkovenko**
Signed-off-by: Anton Gorkovenko <gostom@gmail.com>

#### Is this a new chart
no
#### What this PR does / why we need it:
Update api versions to support k8s 1.16
#### Which issue this PR fixes
no
#### Special notes for your reviewer:
no
#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by antongorkovenko** [comment]
@frankreno @flah00 @darend @bendrucker please review

File: stable/sumologic-fluentd/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sumologic-fluentd
-version: 1.1.1
+version: 1.2.0
```
Comment (bendrucker) [inline stable/sumologic-fluentd/Chart.yaml]: Shouldn't a change to the supported k8s version result in a major bump?

Comment (antongorkovenko) [reply stable/sumologic-fluentd/Chart.yaml]: updated

**Review by bendrucker — COMMENTED** [review]
Can you also go ahead and specify a `kubeVersion` in the chart?

https://helm.sh/docs/developing_charts/#the-chart-yaml-file

The RBAC resources should also graduate to `rbac.authorization.k8s.io/v1` (k8s@1.8).

**Comment by antongorkovenko** [comment]
> Can you also go ahead and specify a `kubeVersion` in the chart?
> 
> https://helm.sh/docs/developing_charts/#the-chart-yaml-file
> 
> The RBAC resources should also graduate to `rbac.authorization.k8s.io/v1` (k8s@1.8).

updated

File: stable/sumologic-fluentd/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 apiVersion: v1
 name: sumologic-fluentd
-version: 1.1.1
+version: 2.0.0
 appVersion: 2.3.0
+kubeVersion: "^1.10.0-0"
```
Comment (bendrucker) [inline stable/sumologic-fluentd/Chart.yaml]:
```suggestion
kubeVersion: ">= 1.10"
```

🙂

Comment (bendrucker) [reply stable/sumologic-fluentd/Chart.yaml]: Otherwise LGTM

Comment (antongorkovenko) [reply stable/sumologic-fluentd/Chart.yaml]: ok

Comment (antongorkovenko) [reply stable/sumologic-fluentd/Chart.yaml]: fyi: it caused pull-charts-e2e fail
restored

**Comment by antongorkovenko** [comment]
@bendrucker all done and tested

**Comment by antongorkovenko** [comment]
@bendrucker @flah00 any updates?

**Pull request merged**